tools/provision

```diff
@@ -62,7 +62,7 @@ if [ "$failed" -ne 0 ]; then
     echo "* Logs are here: zulip/var/log/provision.log"
     echo -e "$ENDC"
     exit "$failed"
-elif [ "$VIRTUAL_ENV" != "/srv/zulip-py3-venv" ] && [ -z "${CIRCLECI}${SKIP_VENV_SHELL_WARNING}" ]; then
+elif [ "$VIRTUAL_ENV" != "/srv/zulip-py3-venv" ] && [ -z "${GITHUB_ACTIONS}${SKIP_VENV_SHELL_WARNING}" ]; then
```

Comment:
I think this change might be better to remove the variable, unless we were seeing this error in the logs (in which case that change should be its own commit, since $CIRCLECI was already empty on GitHub Actions systems.